File: fsharp-functions/OpenAPI/src/model/ResponseTimeFilterResult.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.ResponseTimeFilterLocation

module ResponseTimeFilterResult = 

  //#region ResponseTimeFilterResult

  [<CLIMutable>]
  type ResponseTimeFilterResult = {
    [<JsonProperty(PropertyName = "search_id")>]
    SearchId : string;
    [<JsonProperty(PropertyName = "locations")>]
    Locations : ResponseTimeFilterLocation[];
    [<JsonProperty(PropertyName = "unreachable")>]
    Unreachable : string[];
  }
  
  //#endregion
  

File: scala-akka-http-server/src/main/scala/org/openapitools/server/model/ResponseTimeFilterResult.scala
package org.openapitools.server.model


/**
 * @param searchId  for example: ''null''
 * @param locations  for example: ''null''
 * @param unreachable  for example: ''null''
*/
final case class ResponseTimeFilterResult (
  searchId: String,
  locations: Seq[ResponseTimeFilterLocation],
  unreachable: Seq[String]
)

